packages/uploadthing/src/sdk/index.ts

```diff
@@ -111,3 +111,53 @@ export const listFiles = async () => {
 
   return json.files;
 };
+
+/**
+ * Rename a file in UploadThing storage.
+ *
+ * type Update = {
+ *   fileKey: string;
+ *   newName: string;
+ * };
+ *
+ * @param {Update | Update[]} updates
+ *
+ * @example
+ * await renameFile({ fileKey: "2e0fdb64-9957-4262-8e45-f372ba903ac8_image.jpg", newName: "new_image.jpg" });
+ *
+ * @example
+ * await renameFile([{ fileKey: "2e0fdb64-9957-4262-8e45-f372ba903ac8_image.jpg", newName: "new_image.jpg" }, { fileKey: "1649353b-04ea-48a2-9db7-31de7f562c8d_image2.jpg", newName: "new_image2.jpg" }]);
+ *
+ */
+
+type Rename = {
+  fileKey: string;
+  newName: string;
+};
+
+export const renameFile = async (updates: Rename | Rename[]) => {
```

Comment:
would it make sense to call this `updateFile` instead for future proofing being able to update other attributes than just the name?

Reply:
I considered this, but figured we can always add an `updateFile()` in the future, and if all it does now is allow renaming files, it is a bit confusing to name it `updateFile()`